src/components/Range.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  label {
+    margin-right: 10px;
+  }
+  output {
+    background: white;
+    margin: 10px;
+    padding: 6px 10px;
+    border-radius: 6px;
```

Comment:
We also have a global `borderRadius` in our `theme.js`.

Reply:
done